types/lib/hooks.d.ts

```diff
@@ -97,11 +97,17 @@ export class Hooks<
   /**
    * A similar dummy variable that doesn't exist on the real object. Do not
    * try to access this in real code.
+   *
+   * @deprecated This property will become a Symbol in v7 to prevent collisions.
```

Comment:
Maybe add an expiring TODO for this?

Reply:
Or do you think marking it as deprecated is enough for us to remember to remove it when we get closer to the v7 betas?

Reply:
Done, added a TODO (although it won't be triggered because /types is ignored by eslint)